server/etcdserver/cluster_util.go

```diff
@@ -211,7 +212,15 @@ func allowedVersionRange(downgradeEnabled bool) (minV *semver.Version, maxV *sem
 		// Todo: handle the case that downgrading from higher major version(e.g. downgrade from v4.0 to v3.x)
 		maxV.Minor = maxV.Minor + 1
 		minV = &semver.Version{Major: maxV.Major, Minor: maxV.Minor}
+		return minV, maxV
 	}
+
+	// if unsafeDowngrade is enabled, and one minor version down
+	// safe to not fail (e.g., local version 3.4, cluster version 3.5)
```

Comment:
maybe better to say 
"if unsafeDowngrade is enabled, allow one minor version down"?
here is more about defining the max diff version which is different than the check statement above, which is to check whether it's safe to run or fail?

Reply:
yeah, good point. Will change it.